database/migrations/20230824105507_create_keywords_table.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE "keywords" (
+     "id" uuid DEFAULT uuid_generate_v1(),
+     "user_id" uuid NOT NULL,
+     "keyword" text NOT NULL,
+     "status" text NOT NULL DEFAULT 'pending'::text,
+     "links_count" int8 NOT NULL,
```

Comment:
It seems a little bit dangerous to use an `int8` for that number... 🤔 (same for `non_adword_links_count`). I know we ""normally"" don't expect more than 20 results on a page (maybe with 10 additional ads links on the side), but as long as using something bigger (e.g. int16 or int32) does not have any impactful performance issue, I usually recommend to not shrink the allocation too much. This can avoid future bugs for very negligible/invisible impact on performances.

<img width="683" alt="image" src="https://github.com/markgravity/golang-ic/assets/77609814/d51169b1-68b2-4b64-b9e8-e30d1f1dc726">

Reply:
@malparty we only have these on postgre 🤔 
![CleanShot 2023-08-28 at 16 12 01](https://github.com/markgravity/golang-ic/assets/17875522/c970c655-6464-48e6-9f0b-4c2a7b605c1a)

Reply:
And the current int8 equals to bigint 🤔

Reply:
My suggestion is to use `integer` instead of `int4` because `int4` is an alias to `integers` and `integers` is more generic. See: https://github.com/nimblehq/varun-api/blob/develop/database/migrations/20220125011811_create_delivery_reports.sql#L15